src/NovelRT/DotNet/RuntimeService.cpp

```diff
@@ -94,7 +94,7 @@ namespace NovelRT::DotNet {
       if (result != 0)
       {
         _logger.logError("Failed to locate hostfxr: {}", result);
-        throw std::runtime_error("Failed to locate hostfxr");
+        throw Exceptions::FileNotFoundException("Failed to locate hostfxr");
```

Comment:
This should just be a standard FileNotFoundException like in C# but it accepts the file path it attempted to access but could not find as the argument. I think that would be nicer. :)